test/controllers/records_controller_test.rb

```diff
@@ -121,6 +121,20 @@ class RecordsControllerTest < ActionController::TestCase
       assert_redirected_to collection_path(id: 1, record_id: record.id)
     end
 
+    it 'displays a failure message when attempting to associate a granule to a collection record that has been closed.' do
+      user = User.find_by(email: 'abaker@element84.com')
+      sign_in(user)
+      stub_urs_access(user.uid, user.access_token, user.refresh_token)
+
+      Record.any_instance.stubs(close!: true)
```

Comment:
You should remove this and use an actual record that is closed.   Check fixtures.

Reply:
Done!